app/grandchallenge/workstations/apps.py

```diff
@@ -11,7 +11,7 @@ def init_default_workstation(*_, **__):
 
     try:
         Workstation.objects.get(slug=settings.DEFAULT_WORKSTATION_SLUG)
-    except ObjectDoesNotExist:
+    except (ObjectDoesNotExist, Workstation.DoesNotExist):
```

Comment:
This should be unnecessary.

Reply:
This was an accidental change stemming from debugging... will undo it!